assets/css/staff.css

```diff
@@ -0,0 +1,40 @@
+h1, h2 {
+    text-align: center;
+}
+
+.staff-list {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.staff-card {
+    display:inline-block;
+    border:1px solid black;
+    padding: 5px 10px;
```

Comment:
From the screenshot in the PR Conversation, it looks like the card could use a bit more padding at the top.